src/lib-components/BlockRemoveSearchFilter.vue

```diff
@@ -0,0 +1,121 @@
+<script setup>
+// Helpers
+import { computed } from 'vue'
+import { useRoute } from 'vue-router'
+import SvgGlyphClose from 'ucla-library-design-tokens/assets/svgs/icon-close.svg'
+import getSectionName from '@/utils/getSectionName'
+
+const props = defineProps({
+  title: {
+    type: String,
+    default: '',
+  },
+  filterType: {
+    type: String,
+    default: '',
+  },
+  index: {
+    type: Number,
+    default: 0,
+  },
+  color: {
+    type: String,
+    default: '', // This will be "visit", "about", "help".
+  },
+},)
+
+const emit = defineEmits(['removeBlockFilter'])
+
+const route = useRoute()
+
+const sectionName = computed(() => {
+  return (
+    props.color
+        || (route.path
+          ? getSectionName(route.path)
+          : 'color-default')
+  )
+})
+
+const classes = computed(() => {
+  return ['block-remove-search-filter', `color-${sectionName.value}`]
+})
+
+function closeBlockFilter() {
+  emit('removeBlockFilter', props.index)
+}
+</script>
+
+<template>
+  <span :class="classes">
+    <a>
+      {{ title }}
+    </a>
+    <button type="button" class="button-close" @click="closeBlockFilter">
+      <SvgGlyphClose class="svg-glyph-close" />
+    </button>
+  </span>
+</template>
+
+<style lang="scss" scoped>
+.block-remove-search-filter {
+    padding: 12px;
+    display: flex;
+    flex-direction: row;
+    border: 1.5px var(--color-border) solid;
+    background-color: #ffffff;
+    border-radius: 4px;
+    align-items: center;
+    width: fit-content;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 100%;
+
+    .button-close {
```

Comment:
Removing outline is not ideal for accessibility. Outline should either be left as is or re-styled. Confer with Axa about this.

Reply:
I resolved this by making the entire component focusable like it should be. 
Good call.